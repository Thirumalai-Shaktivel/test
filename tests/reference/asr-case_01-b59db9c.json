{
    "basename": "asr-case_01-b59db9c",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/case_01.f90",
    "infile_hash": "2ac706c2d84ea4be0c07b3d3a2e0069c067130ae1229844524cd2755",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-case_01-b59db9c.stdout",
    "stdout_hash": "92ace3f44ae8e267cfeb65d03b3bf107a42b733c9c84e09b6c93be36",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}