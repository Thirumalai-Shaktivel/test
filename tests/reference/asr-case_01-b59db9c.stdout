(TranslationUnit (SymbolTable 1 {case_01: (Program (SymbolTable 2 {i: (Variable 2 i Local () Default (Integer 4 []) Source Public)}) case_01 [(= (Var 2 i) (ConstantInteger 4 (Integer 4 []))) (Select (Var 2 i) [(CaseStmt [(ConstantInteger 1 (Integer 4 []))] [(Print () [(Str "1" (Character 8 []))])]) (CaseStmt [(ConstantInteger 2 (Integer 4 []))] [(Print () [(Str "2" (Character 8 []))])]) (CaseStmt [(ConstantInteger 3 (Integer 4 []))] [(Print () [(Str "3" (Character 8 []))])]) (CaseStmt [(ConstantInteger 4 (Integer 4 []))] [(Print () [(Str "4" (Character 8 []))])])] []) (Select (Var 2 i) [(CaseStmt [(ConstantInteger 1 (Integer 4 []))] [(Print () [(Str "1" (Character 8 []))])]) (CaseStmt [(ConstantInteger 2 (Integer 4 [])) (ConstantInteger 3 (Integer 4 [])) (ConstantInteger 4 (Integer 4 []))] [(Print () [(Str "2,3,4" (Character 8 []))])])] [])])}) [])
